src/frontend/components/app-trees/app-trees.ts

```diff
@@ -119,6 +120,38 @@ export class AppTrees {
     this.settingOpened = !this.settingOpened;
   }
 
+  private onFeedbackClick = () => {
+    console.log('Feedback button pressed, expand feedback pane!');
+    this.showFeedbackPane = !this.showFeedbackPane;
+  }
+
+  private onReportIssuesClick = () => {
+    const date = (new Date()).toUTCString();
+    const body = `Augury: ${chrome.runtime.getManifest().version}
+    Date: ${date}
+    OS: ${navigator.platform}
+
+    Demo Test Application:
+    -- Git repository for demo app showing the issue (optional but very helpful for difficult issues.)
+    -- If a code snippet will completely show the issue, please include it.
+
+    Description of issue:
+    -- Include (clipped) screenshot images if possible.
+
+    Angular Version (required): ???
```

Comment:
@riavalon I think we might as well populate the version of angular as per @stevenkampen's feature